index.html

```diff
@@ -46,7 +46,7 @@ <h1 class="top-name">ELDON RATZLAFF</h1>
 <section class="midle-section">
     <h2 class="projects">My Recent Works</h2>
 <!-- proyect 1 -->
-<div class="works-cards">
+<div class="works-cards works-cards1">
     <div class="project1"></div>
     <div class="proyect-foot1">
         <h2 class="proyect-foot1-p">Project name goes here</h2>
```

Comment:
- [ ] Dear @Eldonratzlaff, Good job making the changes requested by the previous reviewer. There are some details still remaining though. Remember being able to translate a design into a webpage is one of the major skills and we need to get our design as close as possible to the given one. some of the detail missing are 
- you can still use more margins between the header section and the work section, Also consider making your text size smaller.
-  You have a horizontal scroll which shows that your project is not responsive enough to shrink and grow based on the screen width.

### Figma design
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206888401-8702fb32-5ed4-45f2-9143-2daacfc54034.png)

### Your design 
![Screenshot 2022-12-11 083717](https://user-images.githubusercontent.com/69077061/206888405-01de02c4-f475-4a31-9b79-d6a7a72dbd98.png)